src/CMakeLists.txt

```diff
@@ -10,3 +10,11 @@ install(PROGRAMS ks8851-set-mac revpi-factory-reset piSerial
 
 install(FILES revpi-factory-reset.sh
   DESTINATION /etc/profile.d)
+
+add_custom_target(lan9500.bin ALL
+  COMMAND xxd -r lan9500.hex lan9500.bin
+  DEPENDS /usr/bin/xxd
+  VERBATIM)
+
+install(FILES lan9500.bin
+  DESTINATION /lib/firmware/revpi)
```

Comment:
I don't think this belongs in the firmware directory.
Maybe /usr/lib/<packagename> or /usr/share/<packagename> ...
/lib/firmware is for firmware, which is normally loaded by the kernel.

Reply:
/usr/share/ looks better to me with referring to FHS: https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s11.html.
So put it in the directory /usr/share/revpi/ ?

Reply:
sounds ok